pom.xml

```diff
@@ -116,6 +121,55 @@
       <artifactId>fusionauth-jwt</artifactId>
       <version>${fusionauth-jwt.version}</version>
     </dependency>
+
+    <!-- Test Dependencies Start-->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>${rest-assured.version}</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest</artifactId>
```

Comment:
We should use assertj https://github.com/assertj/assertj, more actively maintained then hamcrest, has fluent assertions.

Reply:
restassured has inbuilt support for hamcrest assertions hence going that way. From what I know, both assertj and hamcrest have similar functionalities.